src/stories/atoms/Shape/Shape.stories.tsx

```diff
@@ -0,0 +1,44 @@
+// Global Imports
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+// Local Imports
+import { Shape } from 'stories/atoms/Shape/Shape';
+
+export default {
+  title: 'Design System/Atoms/Shape',
+  component: Shape,
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          backgroundColor: '#C4C4C4',
+          width: '100px',
+          height: '100px',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as ComponentMeta<typeof Shape>;
+
+const Template: ComponentStory<typeof Shape> = (args) => <Shape {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  color: '#C4C4C4',
+  shape: 'square',
+};
+
+export const Rectangle = Template.bind({});
+Rectangle.args = {
+  ...Default.args,
```

Comment:
How comfortable are you with this spread syntax?

Reply:
I'm fairly comfortable with this spread syntax in general. It seems very similar to using the spread operator when updating state for class based react components. However, I could use another run down on how storybook is really working here.